ui-lib/src/assets/icons/CheckMark.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+const CheckMarkIcon = ({
```

Comment:
Definitely, that's sad that we didn't start by integrating icons first because as soon as the PR related to the icons will be merged we'll have to pass on components merged before this one to use new icons.